epam_task_02/task_02/my_text_reader/src/main/java/entity/Symbol.java

```diff
@@ -0,0 +1,41 @@
+package main.java.entity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Symbol implements TextComponent {
+    private static TextComponent.TextType type = TextComponent.TextType.SYMBOL;
+    private static Map<Character, Symbol> symbols;
+    private char symbol;
```

Comment:
it is bad practice to name variables the same name with class where it defined. Rename variable `symbol`